app/ts/store/account.ts

```diff
@@ -53,11 +53,25 @@ async function connect() {
 				error = new EthereumJsonRpcError(exception.code, exception.message, exception.data)
 			}
 		}
-		store.value = { status: 'rejected', error }
+		accountStore.value = { status: 'rejected', error }
 	}
 }
 
+async function ensureConnected() {
+ 	try {
+ 		assertsExternalProvider(window.ethereum)
+ 		const provider = new ethers.providers.Web3Provider(window.ethereum)
+ 		const signer = provider.getSigner()
+ 		accountStore.value = { status: 'busy' }
+ 		const address = await signer.getAddress()
+ 		accountStore.value = { status: 'connected', address }
+ 	} catch (exception) {
+ 		accountStore.value = { status: 'disconnected', connect, ensureConnected }
```

Comment:
I just realized this was already declared
```suggestion
		accountStore.value = storeDefaults
```